tests/blockchain.test.js

```diff
@@ -88,12 +88,32 @@ describe('Blockchain class', function() {
     		blockchain = createBlockchainWithTx();
     		blockchain.chain[1].transactions[0].amount = 897397;
     		assert(!blockchain.isChainValid());
-    	});
+		});
 
     	it('should fail when a block has been changed', function(){
     		blockchain = createBlockchainWithTx();
     		blockchain.chain[1].timestamp = 897397;
     		assert(!blockchain.isChainValid());
     	});
-    });
+	});
+	
+	describe("getAllTransactionsForWallet", function(){
+		it("should get all Transactions for a Wallet", function() {
+			const validTx = createSignedTx();
+    		blockchain.addTransaction(validTx);
+    		blockchain.addTransaction(validTx);
+
+			blockchain.minePendingTransactions("b2");
+			blockchain.addTransaction(validTx);
+			blockchain.addTransaction(validTx);
+			blockchain.minePendingTransactions("b2");
+
+			blockchain.getAllTransactionsForWallet("b2").forEach((x) => {
```

Comment:
Could you refactor this with a regular for loop?
As in: `for(const tx of blockchain.getAllTransactionsForWallet())`

Reply:
yes will make